tools/ci_build/github/linux/docker/migraphx-ci-pipeline-env.Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM rocm/pytorch:rocm5.5_ubuntu20.04_py3.8_pytorch_1.13.1
+FROM rocm/pytorch:rocm5.6_ubuntu20.04_py3.8_pytorch_1.13.1
```

Comment:
This Dockerfile is used for CI, we need to update more files to support it. You can refer to this PR(https://github.com/microsoft/onnxruntime/pull/15905) or let this Dockerfile unchanged, there is a draft PR to update CI dockerfile(https://github.com/microsoft/onnxruntime/pull/17076)